src/evaluation/utils/custom_pipelines.py

```diff
@@ -242,3 +242,108 @@ def run(self, **kwargs):
             "labels": inputs[0].get("labels", None),
         }
         return output, "output_1"
+
+
+
+
+class MergeOverlappingAnswers():
```

Comment:
To get better consistency with the deplyment, I believe we should define this function [here ](https://github.com/etalab-ia/piaf-ml/blob/master/deployment/roles/haystack/files/custom_component.py) since this is THE file where all "custom components" are defined across all clients.   

You could then import from `/deployment/roles/haystack/files/custom_component.py` your component here

Reply:
I moved the class `MergeOverlappingAnswers` as well as the function `merge_strings` to `/deployment/roles/haystack/files/custom_component.py` like you suggested.

I took the opportunity to remove the class `JoinDocuments` from `src/evaluation/utils/custom_pipelines.py`, as I noticed that it was a duplicate of `JoinDocumentsCustom` in `/deployment/roles/haystack/files/custom_component.py`